community/slack.md

```diff
@@ -1 +1,69 @@
+
+## Setting up an automatic feed to post from GitHub into Slack:
+* First, grant permission to the Slack app from within GitHub. Then you can customize which Slack channels you want to use in order to track new activity on GitHub.
+* To use this integration, you can provide commands directly in the channel to connect it to a specific feed of events on GitHub. Some useful commands are:
+    * **`/GitHub subscribe pyopensci/python-package-guide issues`** will allow you to subscribe to issues in the python-package-guide channel
+    * **`/GitHub unsubscribe pyopensci/python-package-guide pulls`** will unsubscribe you
+    * **`/GitHub help`** provides you with all of the options in terms of subscribing and unsubscribing
+
+## Slack’s limitations - nonprofit plan
+pyOpenSci currently has a Slack paid plan that is provided at no cost for nonprofits. It allows up to 250 users and unlimited saved messages. While this solution currently meets our needs, it will become a challenge in the future as pyOpenSci continues to grow. The main concern is the loss of message history.
+
+We initially chose Slack as most of our community members already had organizations on Slack, and many did not like Discord. As we look ahead and consider our future needs, it’s worth noting that some of our partners have moved to platforms such as Mattermost and Zulip.
+
+## Slack for pyOpenSci Community Discussion
+Slack is also used for broader pyOpenSci community discussion. However, it is currently invitation-only, which models the same processes used by rOpenSci. An invite-only model is important as pyOpenSci grows because:
+* It allows us to grow carefully and slowly
+* It allows us to create an inclusive environment that is evolving into a self-moderating community. This is significantly more challenging to do in an online community with large influxes of members.
+* Everyone invited to Slack has generally already contributed to pyOpenSci in some way, and therefore have a vested interest in the community upon joining.
+### Who we invite to the community:
+We invite people to Slack who have contributed to pyOpenSci in some way. This can include attending and contributing via a sprint, submitting a package (normally this happens after the package has been accepted, but may occur earlier on an ad-hoc basis. Invites are rarely sent before the review is finished), being a reviewer or editor, contributing to a guidebook, or signing up to become involved. Often if someone signs up we meet with them prior to sending a Slack invitation.
+
+We also occasionally invite people that would be valuable contributors to our community in some way, even if they haven’t already contributed! These might be people who express interest in work we are doing and who want to learn more or get involved.
```

Comment:
We might also invite people who are from partner community and/or are doing work that is mission aligned!